<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>944</width>
    <height>818</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(36, 31, 49);</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>7</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QGraphicsView" name="backgroundView">
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="renderHints">
       <set>QPainter::Antialiasing|QPainter::HighQualityAntialiasing|QPainter::SmoothPixmapTransform|QPainter::TextAntialiasing</set>
      </property>
      <property name="transformationAnchor">
       <enum>QGraphicsView::AnchorUnderMouse</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>944</width>
     <height>22</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>222</red>
         <green>221</green>
         <blue>218</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Shadow">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>222</red>
         <green>221</green>
         <blue>218</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>222</red>
         <green>221</green>
         <blue>218</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Shadow">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>222</red>
         <green>221</green>
         <blue>218</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>222</red>
         <green>221</green>
         <blue>218</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>222</red>
         <green>221</green>
         <blue>218</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Shadow">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>222</red>
         <green>221</green>
         <blue>218</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <widget class="QMenu" name="menuFiles">
    <property name="title">
     <string>Files</string>
    </property>
    <addaction name="Open_Image"/>
    <addaction name="Open_PDF"/>
    <addaction name="Save_file"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
   </widget>
   <addaction name="menuFiles"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Shadow">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Shadow">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Shadow">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="Zoom_In_Out"/>
   <addaction name="Add_Text"/>
   <addaction name="Print"/>
  </widget>
  <action name="actionOpen_image">
   <property name="text">
    <string>Open image</string>
   </property>
  </action>
  <action name="actionOpen_PDF">
   <property name="text">
    <string>Open PDF</string>
   </property>
  </action>
  <action name="Open_Image">
   <property name="text">
    <string>Open image</string>
   </property>
  </action>
  <action name="Open_PDF">
   <property name="text">
    <string>Open PDF</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="Save_file">
   <property name="text">
    <string>Save image</string>
   </property>
  </action>
  <action name="Save_One_Pdf">
   <property name="text">
    <string>Save single PDF</string>
   </property>
  </action>
  <action name="Print">
   <property name="icon">
    <iconset resource="../icons/new.qrc">
     <normaloff>:/icons/icons8-printer-64.png</normaloff>:/icons/icons8-printer-64.png</iconset>
   </property>
   <property name="text">
    <string>File to print</string>
   </property>
  </action>
  <action name="Add_Text">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons/new.qrc">
     <normaloff>:/icons/icons8-text-64.png</normaloff>:/icons/icons8-text-64.png</iconset>
   </property>
   <property name="text">
    <string>Add Text</string>
   </property>
  </action>
  <action name="Zoom_In">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons/new.qrc">
     <normaloff>:/icons/zoom_in.png</normaloff>:/icons/zoom_in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom_In</string>
   </property>
  </action>
  <action name="Zoom_Out">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons/new.qrc">
     <normaloff>:/icons/zoom_out.png</normaloff>:/icons/zoom_out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom_Out</string>
   </property>
  </action>
  <action name="Zoom_In_Out">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons/new.qrc">
     <normaloff>:/icons/icons8-zoom-64.png</normaloff>:/icons/icons8-zoom-64.png</iconset>
   </property>
   <property name="text">
    <string>Zoom_In_Out</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../icons/new.qrc"/>
 </resources>
 <connections/>
</ui>
